public/src/main/java/com/revenuecat/purchases/PurchaserInfo.kt

```diff
@@ -42,8 +43,10 @@ data class PurchaserInfo internal constructor(
     val allExpirationDatesByProduct: Map<String, Date?>,
     val allPurchaseDatesByProduct: Map<String, Date?>,
     val requestDate: Date,
-    internal val jsonObject: JSONObject,
-    internal val schemaVersion: Int,
+    // TODO: make private. Or maybe not?
+    val jsonObject: JSONObject,
+    // TODO: make private
+    val schemaVersion: Int,
```

Comment:
I don't think there's a way to make this private

Reply:
yeah it's read only, but accessible. I think it's fine. If you look into SkuDetails, they also have a public `getOriginalJson`, so I don't think it's bad we expose this type of stuff.